### PR TITLE
Method uses `generics`, `abc property`, and `abc method`

### PR DESCRIPTION
**Pull request by PerisicJere**

### CODE REVIEW

File: library/src/fy_library/mixins/property/generated_fy_py_code/jinja2_templates/method.jinja2 [hunk 1]
```diff
@@ -1,23 +1,24 @@
 class {{ python_class_name.pascal_case }}
-{%- if abstract_property_mixins or  abstract_method_mixins -%}
+{%- if abstract_property_mixins or  abstract_method_mixins or generics_def-%}
```
Comment (vinkobuble) [inline library/src/fy_library/mixins/property/generated_fy_py_code/jinja2_templates/method.jinja2:2]:
```suggestion
{%- if abstract_property_mixins or  abstract_method_mixins or generics_def -%}
```

**Review by vinkobuble — APPROVED** [review]

File: library/tests/fy_py_files/test_fy_py_files/method_with_generics_impl/method_with_generics_impl_fy.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+"""fy
+from .greetings_t import GreetingT, FrenchGreeting, SpanishGreet
+
+
+method greet[GreetingT](greeting: str) -> int using constant:
+    property french_greeting[FrenchGreeting]
```
Comment (vinkobuble) [inline library/tests/fy_py_files/test_fy_py_files/method_with_generics_impl/method_with_generics_impl_fy.py:9]:
```suggestion
    property greeting[FrenchGreeting]
```

File: library/tests/fy_py_files/test_fy_py_files/method_with_generics_impl/abc_property/abc_fy.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+"""fy
+from ..greetings_t import FrenchGreeting
+
+
+property french_greeting: FrenchGreeting
```
Comment (vinkobuble) [inline library/tests/fy_py_files/test_fy_py_files/method_with_generics_impl/abc_property/abc_fy.py:8]:
```suggestion
property greeting[GreetingT]:
```

**Review by vinkobuble — CHANGES_REQUESTED** [review]

File: library/src/fy_library/mixins/property/generated_fy_py_code/jinja2_templates/method.jinja2 (hunk 1 above)
Comment (PerisicJere) [reply library/src/fy_library/mixins/property/generated_fy_py_code/jinja2_templates/method.jinja2:2]: Changed

File: library/tests/fy_py_files/test_fy_py_files/method_with_generics_impl/method_with_generics_impl_fy.py (hunk 2 above)
Comment (PerisicJere) [reply library/tests/fy_py_files/test_fy_py_files/method_with_generics_impl/method_with_generics_impl_fy.py:9]: Changed

File: library/tests/fy_py_files/test_fy_py_files/method_with_generics_impl/abc_property/abc_fy.py (hunk 3 above)
Comment (PerisicJere) [reply library/tests/fy_py_files/test_fy_py_files/method_with_generics_impl/abc_property/abc_fy.py:8]: changed

**Review by vinkobuble — APPROVED** [review]

**Pull request merged**